docs/translations-needed.md

```diff
@@ -252,16 +252,8 @@ If you add or update a translation run `grunt check-translations` to update the
 |                         | , opens subtitles settings dialog                                                   |
 |                         | , opens descriptions settings dialog                                                |
 |                         | , selected                                                                          |
-| ru.json (missing 14)    | Close Modal Dialog                                                                  |
-|                         | Descriptions                                                                        |
-|                         | descriptions off                                                                    |
-|                         | Audio Track                                                                         |
+| ru.json (missing 6 )    | Close Modal Dialog                                                                  |
```

Comment:
"Close Modal Dialog" is shows up as text for the close button on the modal dialog.
You can see this by opening a modal dialog and hovering over the close button:
```js
player.createModal('foo');
```